include/alpaka/atomic/AtomicCudaBuiltIn.hpp

```diff
@@ -82,11 +85,12 @@ namespace alpaka
             //-----------------------------------------------------------------------------
             //! The GPU CUDA accelerator atomic operation.
             //-----------------------------------------------------------------------------
-            template<>
+            template<typename THierarchy>
             struct AtomicOp<
                 op::Add,
                 atomic::AtomicCudaBuiltIn,
-                int>
+                int,
+                THierarchy>
```

Comment:
You will have to do the same for all of the `struct AtomicOp<` in this file to compile correctly.